packages/tizen_rpc_port/lib/src/port.dart

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:flutter/services.dart';
+import 'package:tizen_interop/6.5/tizen.dart';
+
+import 'parcel.dart';
+
+/// Enumeration for RPC port types.
+enum PortType {
+  /// Main channel to communicate.
+  main,
+
+  /// Sub channel for callbacks.
+  callback,
+}
+
+/// The class that proxy and stub can use to communicate with each other.
+class Port {
+  /// Constructor of this class.
+  Port(this._handle);
+
+  late final rpc_port_h _handle;
+
+  /// The native handle of this port.
+  rpc_port_h get handle => _handle;
+
+  /// Sends a parcel to the connected app.
+  void send(Parcel parcel) {
+    final int ret = tizen.rpc_port_parcel_send(parcel.handle, _handle);
+    if (ret != 0) {
+      throw PlatformException(
+        code: ret.toString(),
+        message: tizen.get_error_message(ret).toDartString(),
+      );
+    }
+  }
+
+  /// Receives a parcel from connected app.
+  /// This api should be used only guaranteed receive something after send().
+  // Future<Parcel> receive() async {
+  //   final Uint8List raw = await _methodChannel.portReceive(this);
+  //   return Parcel.fromRaw(raw);
```

Comment:
If the Parcel.send(Port port) method is added, these methods can be removed.
Please check the following link:
https://github.com/Samsung/TizenFX/blob/master/src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Parcel.cs

Please consider it.

Reply:
I moved `send()` method to parcel.
Thank you :)